lib/StartInfo.cpp

```diff
@@ -64,7 +64,7 @@ std::string StartInfo::getCampaignName() const
 void LobbyInfo::verifyStateBeforeStart(bool ignoreNoHuman) const
```

Comment:
Interesting if we can return an error enum instead of throwing it. So that thing we expect we return and things we do not expect we throw

Reply:
there are few places using this function but exception catching performed only in one place. It requires the refactoring, but I would prefer to postpone it and minimize changes for the release.